de-lib/src/main/java/org/iplantc/de/diskResource/client/views/grid/DiskResourceColumnModel.java

```diff
@@ -49,6 +49,8 @@
                                                                                   CopyPathSelected.HasCopyPathSelectedEventHandlers {
 
 
+    private static List<ColumnConfig<DiskResource, ?>> list;
```

Comment:
Why did you a local variable of a static method get promoted to a private static field?  It's not being used anywhere else. Also, the original static method overwrites it each time, so it's not even acting like a cached copy.

Reply:
Good catch. I started with this approach and realized later i don't need get `ColumnConfig` from here. This change is not required.